features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -65,6 +101,8 @@ internal class JankStatsActivityLifecycleListener(
                 activeWindowsListener[window] = jankStats
```

Comment:
Not related with this PR but maybe we should add a task here to not use anymore the `window` as a key here. See the previous problems we had with using the activity as a key for view scopes.